startupscript/git-setup.sh

```diff
@@ -57,13 +57,13 @@ ${RUN_AS_LOGIN_USER} "mkdir -p '${WORKBENCH_GIT_REPOS_DIR}'"
 # Keep this as last thing in script. There will be integration test for git cloning (PF-1660). If this is last thing, then
 # integration test will ensure that everything in script worked.
 
-# This loop replaces the logic of "/usr/bin/wb git clone --all", which currently does not work without
+# This loop replaces the logic of "wb git clone --all", which currently does not work without
 # Google Application Default Credentials being available.  This emulates the behavior of the CLI
 # command, continuing with an error message when an individual repo cannot be cloned.
 
 # shellcheck disable=SC2164
 pushd "${WORKBENCH_GIT_REPOS_DIR}"
-${RUN_AS_LOGIN_USER} "/usr/bin/wb resource list --type=GIT_REPO --format json" | \
+${RUN_AS_LOGIN_USER} "${WORKBENCH_INSTALL_PATH} resource list --type=GIT_REPO --format json" | \
```

Comment:
```suggestion
${RUN_AS_LOGIN_USER} "${WORKBENCH_INSTALL_PATH}" resource list --type=GIT_REPO --format json | \
```

Reply:
You should just need to quote the `${WORKBENCH_INSTALL_PATH}` here.
Technically, you don't need to quote it, since it is a singe token (`/usr/bin/wb`, not `/usr/bin/my dir/wb`), but it is a good practice to quote it.

`${RUN_AS_LOGIN_USER}` is an exception here and should NOT be quoted.